README.md

```diff
@@ -73,3 +75,13 @@ The server will start locally and on ngrok. The restaurants will not load becaus
 
 * [Here is the City's list of layers](http://maps.huntsvilleal.gov/arcgis/rest/services/Layers), each of which provides a REST endpoint from which we can query data. 
 * [Here's an example of a URL where you might query data](http://maps.huntsvilleal.gov/arcgis/rest/services/Layers/ParkLocations/MapServer/0/query).
+
+# Developers
+
+Instead of building the server and launching the server via make run, an alternative method to start the server is:
+>export GOOGLE_PLACES_API_KEY=supersecretkey
+>cd Frontier
+>npm start
+
+This eliminates the required build step allowing for faster debug cycles.
```

Comment:
Can you accept this change and I'll fix it with my next issue on work flow?
On May 31, 2015 9:47 PM, "Chris Heath" <notifications@github.com> wrote:

> In README.md
> <https://github.com/codeforhuntsville/Frontier/pull/42#discussion_r31398557>
> :
>
> > @@ -73,3 +75,13 @@ The server will start locally and on ngrok. The restaurants will not load becaus
> >
> >  * [Here is the City's list of layers](http://maps.huntsvilleal.gov/arcgis/rest/services/Layers), each of which provides a REST endpoint from which we can query data.
> >  * [Here's an example of a URL where you might query data](http://maps.huntsvilleal.gov/arcgis/rest/services/Layers/ParkLocations/MapServer/0/query).
> > +
> > +# Developers
> > +
> > +Instead of building the server and launching the server via make run, an alternative method to start the server is:
> > +>export GOOGLE_PLACES_API_KEY=supersecretkey
> > +>cd Frontier
> > +>npm start
> > +
> > +This eliminates the required build step allowing for faster debug cycles.
>
> Specifically, this avoids having to build the whole docker image and
> instead simply runs the project.
>
>  npm start
>
> is still running webpack which is technically a build step.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeforhuntsville/Frontier/pull/42/files#r31398557>.
>